tests/integration/test_alerts.py

```diff
@@ -24,16 +24,13 @@ def test_alerts_search_command_returns_success_return_code(
 
 
 @pytest.mark.integration
-@pytest.mark.parametrize(
```

Comment:
Was seeing random send-to tests fail all the time.
I tried 3 things to fix it with no avail:

1.) sleeping everywhere possible, even large amounts of time

2.) Using different ports per instance of the ncat server

The error is that the connection is refused or breaks somehow.

Reply:
I know at one point we said that testing both TCP and UDP is superfluous so I opted to remove these tests for now...

Reply:
If anyone knows why these tests fail like this, please enlighten me and I can try to work it out!

Reply:
Do we need to install nmap in the image?

```
FileNotFoundError: [Errno 2] No such file or directory: 'ncat': 'ncat'
```

Reply:
Yes, somehow, I forgot about that, will try to do this now

Reply:
The failing tests I was describing was happening locally

Reply:
Yeah, they're intermittently failing for me to locally. Investigating.

Reply:
If the `UDP` test executes before the `TCP`, it seems to pass

Reply:
Somehow the tests are still failing here, so weird. They seem to pass consistently locally now at least...

Reply:
It seems it either has to be `yield_fixtures` or there is something else going on...